draft-ietf-tls-rfc8446bis.md

```diff
@@ -2432,7 +2439,7 @@ For the curves secp256r1, secp384r1, and secp521r1,
 peers MUST validate each other's public value Q by ensuring
 that the point is a valid point on the elliptic curve.
 The appropriate validation procedures are defined in Appendix D.1 of {{ECDP}}
-and alternatively in Section 5.6.2.3 of {{?KEYAGREEMENT=DOI.10.6028/NIST.SP.800-56Ar2}}.
```

Comment:
You could use the same DOI method to get SP 800-56Ar3. So why not?

Reply:
Yes, after all, why not?